### ISSUE TITLE
Header text color

### ISSUE THREAD

**Issue by CodingCarlos** [tomloprod/cordova-plugin-headercolor#3]
I don't know how dificult could be, but would be great being able to change also the text color.

**Comment by tomloprod** [comment]
Hola @CodingCarlos,

Is not possible to force the text color, it is auto generated (*white or black*) based on the **contrast ratio**. 

If the *contrast ratio* between your header color and white is above 3 it will use the light color. If not, black.

You can check the contrast ratio of your header color here: http://leaverou.github.io/contrast-ratio/

Also you can see colors of the material design guidelines here (*with black and white text depending of the contrast ratio*): https://material.io/guidelines/style/color.html#color-color-palette

Regards,

**Comment by CodingCarlos** [comment]
Thank you very much! 

It's really usefull to know that, and the resource is so cool. 

Great plugin, and great work, you rock!
